hack/make/.integration-daemon-stop

```diff
@@ -11,7 +11,9 @@ if [ ! "$(go env GOOS)" = 'windows' ]; then
 			echo >&2 "warning: PID $pid from $pidFile had a nonzero exit code"
 		fi
 		root=$(dirname "$pidFile")/root
-		umount "$root" || true
+		if [ -d "$root" ]; then
+			umount "$root" || true
```

Comment:
in practice, this only finds the `docker.pid` of the daemon that was started by `.integration-daemon-start`, but that daemon uses the default (`/var/lib/docker`) as path, so nothing gets unmounted. I think this will only do something if a daemon was not killed as part of a test

Reply:
I'd rephrase as
```shell
test -d "$root" && umount "$root" || true
```

Reply:
and even then you can still get "not mounted" error (afaik there's no way to work around it)

Reply:
Ah, yes, that'd work

I realize it won't fix all situations (always gonna be a race), but currently, it _always_ prints this message

Reply:
hm, so shellcheck doesn't like your approach, and would require a `//nolint` for that; https://github.com/koalaman/shellcheck/wiki/SC2015

I'll keep the current approach (although it's slightly more verbose)

Reply:
Ahh, this is the case when shellcheck is making things worse (and yes, I would not want to add `// nolint`).

Let it stay as it is.